<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionWindow</class>
 <widget class="QWidget" name="ActionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>816</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="deviceLabel">
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="deviceCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="parameterLabel">
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="parameterCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="stateLabel">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="focusButton">
     <property name="text">
      <string>Click to Focus</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
